sljit_src/sljitNativePPC_common.c

```diff
@@ -446,19 +446,21 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 	struct sljit_jump *jump;
 	struct sljit_const *const_;
 	struct sljit_put_label *put_label;
+	sljit_u32 pad_size = 0;
+#if defined(SLJIT_INDIRECT_CALL) && SLJIT_INDIRECT_CALL
+	sljit_u32 trampoline_size = sizeof(struct sljit_function_context);
+#else /* !SLJIT_INDIRECT_CALL */
+	sljit_u32 trampoline_size = 0;
+#endif /* SLJIT_INDIRECT_CALL */
 
 	CHECK_ERROR_PTR();
 	CHECK_PTR(check_sljit_generate_code(compiler));
 	reverse_buf(compiler);
 
-#if (defined SLJIT_INDIRECT_CALL && SLJIT_INDIRECT_CALL)
 #if (defined SLJIT_CONFIG_PPC_64 && SLJIT_CONFIG_PPC_64)
-	compiler->size += (compiler->size & 0x1) + (sizeof(struct sljit_function_context) / sizeof(sljit_ins));
-#else
-	compiler->size += (sizeof(struct sljit_function_context) / sizeof(sljit_ins));
-#endif
+	pad_size = (compiler->size & 0x1) * sizeof(sljit_ins);
```

Comment:
This is a different code than the original. Indirect calls may or may not be present regardless of 32/64 bit.

Reply:
but the padding is only needed for 64bit.